src/main/java/me/botsko/prism/Prism.java

```diff
@@ -712,6 +749,45 @@ public void onDisable() {
             prismDataSource.dispose();
         }
         log("Closing plugin.");
+        for (Handler handler : prismLog.getHandlers()) {
+            handler.close();
+        }
         super.onDisable();
     }
+
+    private static class PrismFileHandler extends FileHandler{
+
+        public PrismFileHandler(File file) throws IOException, SecurityException {
+            super(file.toString());
+            setFormatter(new SimpleFormatter(){
```

Comment:
Codacy found an issue: [WhitespaceAround: '{' is not preceded with whitespace.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5911179)